packages/ember-views/lib/mixins/text_support.js

```diff
@@ -3,11 +3,14 @@
 @submodule ember-views
 */
 
+import Ember from "ember-metal/core"; // FEATURES
 import { get } from "ember-metal/property_get";
 import { set } from "ember-metal/property_set";
 import { Mixin } from "ember-metal/mixin";
 import TargetActionSupport from "ember-runtime/mixins/target_action_support";
 
+var isEnabled = Ember.FEATURES.isEnabled;
```

Comment:
You should not do this because it will prevent defeaturify from properly stripping out the code for production builds.